beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,193 @@
+# Inline nn.Sequential Example:
+# ----------------------------
+#
+
+import os
+import torch
+import torch.nn as nn
+import torch.onnx as onnx
+from torch.utils.data import DataLoader
+from torchvision import datasets, transforms
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+# model
+model = nn.Sequential(
+        nn.Flatten(),
+        nn.Linear(28*28, 512),
+        nn.ReLU(),
+        nn.Linear(512, 512),
+        nn.ReLU(),
+        nn.Linear(512, len(classes)),
+        nn.Softmax(dim=1)
+    ).to(device)
+    
+print(model)
+
+#############################################
+# Class nn.Module Example:
+# --------------------------
+#
+
+
+class NeuralNetwork(nn.Module):
+    def __init__(self):
+        super(NeuralNetwork, self).__init__()
+        self.flatten = nn.Flatten()
+        self.layer1 = nn.Linear(28*28, 512)
+        self.layer2 = nn.Linear(512, 512)
+        self.output = nn.Linear(512, 10)
+
+    def forward(self, x):
+
+        x = self.flatten(x)
+        x = F.relu(self.layer1(x))
+        x = F.relu(self.layer2(x))
+        x = self.output(x)
+        return F.softmax(x, dim=1)
+model = NeuralNetwork().to(device)
+    
+print(model)
+
+
+#############################################
+# Get Device for Training
+# -----------------------
+# Here we check to see if `torch.cuda <https://pytorch.org/docs/stable/notes/cuda.html>`_ is available to use the GPU, else we will use the CPU. 
+#
+# Example:
+#
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+##############################################
+# __init__
+# -------------------------
+#
+# The ``init`` function inherits from ``nn.Module`` which is the base class for 
+# building neural network modules. This function defines the layers in your neural network
+# then it initializes the modules to be called in the ``forward`` function.
+# 
+
+##############################################
+# The Model Module Layers
+# -------------------------
+#
+# Lets break down each model layer in the FashionMNIST model.
+#
+
+##################################################
+# `nn.Flatten <https://pytorch.org/docs/stable/generated/torch.nn.Flatten.html>`_ 
+# -----------------------------------------------
+#
+# First we call nn.Flatten to reduce tensor dimensions to one.
+#
+# From the docs:
+# ``torch.nn.Flatten(start_dim: int = 1, end_dim: int = -1)``
+#
+# Here is an example using one of the training_data set items:
+tensor = training_data[0][0]
+print(tensor.size())
+
+model = nn.Sequential(
+    nn.Flatten()
+)
+flattened_tensor = model(tensor)
+flattened_tensor.size()
+
+##############################################
+# `nn.Linear <https://pytorch.org/docs/stable/generated/torch.nn.Linear.html>`_ to add a linear layer to the model.
+# -------------------------------
+#
+# Now that we have flattened our tensor dimension we will apply a linear layer 
```

Comment:
A better description of the Linear layer is `a module that applies a linear transformation on the input using it's stored weights and biases`. The module itself doesn't calculate/learn the w & b

Reply:
I will fix this wording.